scripts/auth_init.sh

```diff
@@ -0,0 +1,5 @@
+ #!/bin/sh
+
```

Comment:
This is going to end up re-calling pip install even for users that aren't using authentication. This was a failing of my branch as well that you inherited.

I suggest adding an additional check here for whether USE_AUTHENTICATION is enabled before running pip install.

I'm open to other ideas for how to keep this a complete no-op for the no-auth user.

Reply:
We might also consider -q to reduce the pip output overall, but I haven't played with it to see if it removes necessary output for debugging:
https://pip.pypa.io/en/stable/cli/pip/#cmdoption-q